tutorials/templates/detail.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base/base.html" %}
+{% load bootstrap3 %}
+
+{% block main-content-body %}
+{% load staticfiles %}
+
+    <a href="{% url 'edit_tutorial' tutorial_id=tutorial.id %}"> Bearbeiten </a>
+    <a href="{% url 'delete_tutorial' tutorial_id=tutorial.id %}"> Löschen </a>
+    <span>Tutorial: {{tutorial.id}}</span>
```

Comment:
This looks like a debug display

Reply:
Thank you for the review :) @MGlauer 

The edit function does not yet distinguish between static and dynamic tutorials. Static tutorials (tutorials from examples repo.) cannot yet be edited here. I would like to implement this in a following PR to reduce the number of commits per PR. 

I do not know exactly what you mean by debug display?

Reply:
Then I would suggest to disable the edit-button with a mouse-over note to avoid confusion. Static tutorials should be edited. As far as I understand the tutorials are imported from the jupyter notebooks on github. I guess this would be the proper place for editing - not the OEP.

Reply:
Regarding the "debug output": IDs are normally just a number used by data management to identify objects. These numbers do not really hold any interesting information for users and, therefore, should not be displayed.

Reply:
That's the way I wanted to implement it :) .

Oh oops that´s true I didn't realize I put the id as output there. I will fix that.

Reply:
The title of a tutorial is now displayed correctly in the detail-view. 

I decided to change the behavior of the edit-"button" and removed the delete button from all static tutorials to avoid the error but leave the frontend work to @johannwagner .